### DISCUSSION TITLE
License type

### DISCUSSION THREAD

**Discussion by rbevin777** [ThrowTheSwitch/Ceedling#788] [Q&A] [answered]
Hi, 

I'm just wondering what exact license you guys use for this? Is it MIT, a modified version of the MIT license or is it something else altogether? 

Thanks

**Comment by Letme** [comment]
If you read it it is not MIT, so you can say modified MIT or just something custom. Most important thing is that you include acknowledgement in your documentation - so nothing really prohibitive.

**Reply by rbevin777** [reply]
No problem, it's not very clear. There are a lot of license types. The only reason I ask is because I'd like to add this software to a list of C tools https://github.com/oz123/awesome-c and I want to make sure the correct license is attributed.

**Reply by Letme** [reply, marked as answer]
Indeed there are, kinda general problem with open source.
I don't know if any maintainers will join the discussion, but there are so many contributions by now, that license change seems highly unlikely.

**Reply by rbevin777** [reply]
No problem, thanks for your response.

**Comment by mkarlesky** [comment]
The clause that caused the license to be modified MIT was removed a good while ago.
